provider/docker.go

```diff
@@ -270,7 +270,12 @@ func (provider *Docker) getIPAddress(container dockertypes.ContainerJSON) string
 			}
 		}
 	}
-	for _, network := range container.NetworkSettings.Networks {
+	for networkName, network := range container.NetworkSettings.Networks {
```

Comment:
I'm pretty sure it's more reliable to look at `container.HostConfig.NetworkMode` to detect the `host` network mode :angel: